sklearn/model_selection/_split.py

```diff
@@ -860,13 +886,13 @@ def get_n_splits(self, X=None, y=None, groups=None):
 
         Parameters
         ----------
-        X : object, optional
+        X : object
             Always ignored, exists for compatibility.
 
-        y : object, optional
```

Comment:
Why did you change this?

Reply:
Everywhere else the "Always ignored, exists for compatibility." statement appears in this file does not include the `, optional`, so I dropped it here to keep it consistent.

Reply:
I would argue that with "optional" it's clearer but I don't have a strong opinion. Looks good otherwise.